### PR TITLE
Frontend: Error on HistoryView

### LINKED ISSUE #576: Frontend: Error on HistoryView
After #570 there is an error when loading `HistoryView`:
```
VM520 react_devtools_backend.js:4061 TypeError: Cannot read properties of undefined (reading 'id')
    at oc.doResume (save-frontend.js:1:1554010)
    at Oa.local$closure$getData (save-frontend.js:1:1555730)
    at Oa.doResume (save-frontend.js:1:1513125)
    at Oa.Cn.resumeWith_tl1gpc$ (save-frontend.js:1:585805)
    at Nn.Ln.run (save-frontend.js:1:893679)
    at fr.mr.process (save-frontend.js:1:923778)
    at save-frontend.js:1:923355
```

### PR DESCRIPTION
**Pull request by Cheshiriks**
What's done:
* fixed error on HistoryView
* fixed deserialize for LocalDateTime
Closes #576

### CODE REVIEW

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/utils/LocalDateTimeConfig.kt [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    serializersModule = SerializersModule {
+        contextual(LocalDateTime::class, LocalDateTimeSerializer)
+    }
+}
+
+@Configuration
+class LocalDateTimeConfig {
+
+    @Bean
+    fun kotlinSerializationJsonEncoder() = KotlinSerializationJsonEncoder()
```
Comment (petertrr) [inline save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/utils/LocalDateTimeConfig.kt:33]: We should also inject `json` here

File: save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/utils/LocalDateTimeSerializer.kt [hunk 2]
```diff
@@ -9,13 +8,15 @@ import kotlinx.serialization.descriptors.PrimitiveKind
...
+import kotlinx.serialization.serializer
 
 object LocalDateTimeSerializer : KSerializer<LocalDateTime> {
     override val descriptor = PrimitiveSerialDescriptor("timestamp", PrimitiveKind.LONG)
-    override fun deserialize(decoder: Decoder): LocalDateTime = LocalDateTime.ofInstant(
-        Instant.ofEpochSecond(decoder.decodeLong()),
-        ZoneOffset.UTC
-    )
+
+    override fun deserialize(decoder: Decoder): LocalDateTime {
```
Comment (petertrr) [inline save-cloud-common/src/jvmMain/kotlin/org/cqfn/save/utils/LocalDateTimeSerializer.kt:16]: Pls fix `serialize` method too

**Review by petertrr — CHANGES_REQUESTED** [review]
Please fix `serialize` and put `sarif4k` submodule back

File: save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/utils/LocalDateTimeConfig.kt (hunk 1 above)
Comment (Cheshiriks) [reply save-preprocessor/src/main/kotlin/org/cqfn/save/preprocessor/utils/LocalDateTimeConfig.kt]: Fixed

**Review by petertrr — APPROVED** [review]

**Pull request merged**